csss-site/src/csss/templates/csss/header.html

```diff
@@ -91,14 +91,13 @@
 						{% endif %}
 						<div class="navbar-dropdown">
 							<a href="{{URL_ROOT}}events/regular_events" class="navbar-item">Regular Events</a>
+							<a href="{{URL_ROOT}}events/tech_fair/2022" class="navbar-item">Tech Fair 2022</a>
```

Comment:
this does not exist?

Reply:
Oops... part of my next PR. Should be out today.